### DISCUSSION TITLE
Custom rendering (e.g. math)

### DISCUSSION THREAD

**Discussion by AustenLamacraft** [giscus/giscus#272] [Q&A] [answered]
For technical content I'd really like to be able to render LaTeX math. I see that rendering is handled by GitHub at the moment, and they don't support LaTeX. Is there any way around this? As I understand it, I wouldn't be able to manipulate the iframe returned by giscus so perhaps some self-hosted solution is inevitable?

Thanks for a great project!

**Comment by laymonage** [comment, marked as answer]
Yeah, unfortunately I don't plan on customizing the renderer or modifying the user input before submitting it to GitHub. The goal is to embed GitHub Discussions to your website, so I try to make sure that the comments render the same way on your website and on GitHub. You'll have to self-host the app if you want such modifications.

Meanwhile, you can also check out this trick:

https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b

Also, this extension might work but I haven't tried it:

https://chrome.google.com/webstore/detail/mathjax-plugin-for-github/ioemnmodlmafdkllaclgeombjnmnbima
